### PR TITLE
AND-28 - Send Action ID to server with button clicks on Notes.

### PR DESCRIPTION
**Pull request by skykelsey**

### CODE REVIEW

File: apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/model/TextModalInteraction.java [hunk 1]
```diff
@@ -24,8 +24,8 @@
 	public static final String EVENT_NAME_DISMISS = "dismiss";
 	public static final String EVENT_NAME_INTERACTION = "interaction";
 
-	public static final String EVENT_KEY_LABEL = "label";
```
Comment (msaffitz) [inline apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/model/TextModalInteraction.java]: was this just not used?

**Comment by msaffitz** [comment]
:+1: , one question.   otherwise looks good

**Comment by CodeMonkeySteve** [comment]
:+1:

File: apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/model/TextModalInteraction.java (hunk 1 above)
Comment (skykelsey) [inline apptentive/src/com/apptentive/android/sdk/module/engagement/interaction/model/TextModalInteraction.java]: I switched to using the already present key with the same value from the Target object.

**Comment by pkamb** [comment]
:+1:

**Pull request merged**